src/com/eclectide/intellij/whatthecommit/WhatTheCommitAction.java

```diff
@@ -60,19 +61,39 @@ public void actionPerformed(AnActionEvent e) {
         }
     }
 
+    private String readUrlContent(String urlString) {
+        HttpConfigurable httpConfigurable = HttpConfigurable.getInstance();
+
+        HttpURLConnection connection = null;
+
+        try {
+            connection = httpConfigurable.openHttpConnection(urlString);
+
+            String text = StreamUtil.readText(connection.getInputStream(), "UTF-8");
+            return text.trim();
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        } finally {
+            if (connection != null) {
+                connection.disconnect();
+            }
+        }
+
+        return null;
```

Comment:
Thanks for your contribution! 
While testing your change, IntelliJ aborts the build because this line is not reachable (it exits either in line 73 or 75). Could you please remove this line?

![image](https://user-images.githubusercontent.com/3101914/93497920-5db31880-f911-11ea-88d8-3ea9c703b04e.png)

Reply:
Hmm this didn't fail the build on my side, but I do get the same response.
I removed the line and also cleaned another statement in the `getCheckinPanel()` on which the code inspection gave a warning.